backends/graphs/controls.cpp

```diff
@@ -67,6 +67,18 @@ class EdgeSwitch : public EdgeTypeIface {
     const IR::Expression *labelExpr;
 };
 
+class EdgeSelect : public EdgeTypeIface {
```

Comment:
This is not used in this file, so I'm not sure why it is defined here

Reply:
This will be removed. Remnant code from my initial experimentation with code.